# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/CanonicalBrowsingContext.cpp
# Commit: 41de9ac734d6
# Full Hash: 41de9ac734d69d754ea3f13db6933343b62ea1b8
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-07-21 09:42:41
# Regressor Bug: 1646899
# File Overlap Count: 1
# Description:
#   Bug 1646899 - P4: Handle object & embed via DocumentChannel. r=mattwoodrow,jya
#   
#   Pass internal content policy type to DLL and switch behavior depending on type
#   being loaded. This implementation immediately redirects channel back to the
#   content process for further handling.
# ==============================================================================

diff -r 64e118a38c1d -r 41de9ac734d6 docshell/base/CanonicalBrowsingContext.cpp
--- a/docshell/base/CanonicalBrowsingContext.cpp	Tue Jul 21 01:01:03 2020 +0000
+++ b/docshell/base/CanonicalBrowsingContext.cpp	Tue Jul 21 01:01:05 2020 +0000
@@ -1034,7 +1034,7 @@
   // Ideally in the future we will only start loads from here, and we can
   // just set this directly instead.
   return net::DocumentLoadListener::LoadInParent(this, aLoadState,
-                                                 outerWindowId, aSetNavigating);
+                                                 aSetNavigating);
 }
 
 bool CanonicalBrowsingContext::AttemptSpeculativeLoadInParent(
@@ -1058,8 +1058,7 @@
   // If we successfully open the DocumentChannel, then it'll register
   // itself using aLoadIdentifier and be kept alive until it completes
   // loading.
-  return net::DocumentLoadListener::SpeculativeLoadInParent(this, aLoadState,
-                                                            outerWindowId);
+  return net::DocumentLoadListener::SpeculativeLoadInParent(this, aLoadState);
 }
 
 bool CanonicalBrowsingContext::StartDocumentLoad(